src/include/sof/audio/audio_stream.h

```diff
@@ -118,6 +121,18 @@ static inline enum sof_ipc_frame audio_stream_get_frm_fmt(
 	return buf->frame_fmt;
 }
 
+static inline enum sof_ipc_frame audio_stream_get_fmt_shift(
```

Comment:
Why does this return enum sof_ipc_frame? Probably, it should return int8_t or signed char.

Reply:
copy-paste, will fix, thanks!